docs/topics/Threads.md

```diff
@@ -200,32 +200,40 @@ Messages cannot be sent directly in forum channels.
 
-Threads in a forum channel have the same permissions behavior as threads in a text channel, inheriting all permissions from the parent channel, with one exception: creating a thread in a forum channel only requires the `SEND_MESSAGES` permission.
+> warn
+> `GUILD_MEDIA` channels are in beta and still being actively developed. The API and other technical details are subject to change.
+
+
+### Creating Threads in Forum and Media Channels
+
+Within thread-only channels, threads appear as posts. Threads can be created using the [`POST /channels/<channel_id>/threads`](#DOCS_RESOURCES_CHANNEL/start-thread-in-forum-or-media-channel) endpoint with [slightly different parameters](#DOCS_RESOURCES_CHANNEL/start-thread-in-forum-or-media-channel-jsonform-params) than threads in text channels. For example, when creating threads in a threads-only channel, a message is created that has the same ID as the thread which requires you to pass parameters for both a thread *and* a message.
+
+Threads in thread-only channels have the same permissions behavior as threads in a text channel, inheriting all permissions from the parent channel, with one exception: creating a thread in a threads-only channel only requires the `SEND_MESSAGES` permission.
 
-### Forum Channel Fields
+### Forum and Media Channel Fields
 
-It's worth calling out a few details about fields specific to forum channels that may be important to keep in mind:
+It's worth calling out a few details about fields specific to thread-only channels that may be important to keep in mind:
 
 - The `last_message_id` field is the ID of the most recently created thread in that channel. As with messages, your app will not receive a `CHANNEL_UPDATE` event when the field is changed. Instead clients should update the value when receiving [Thread Create](#DOCS_TOPICS_GATEWAY_EVENTS/thread-create) events.
 - The `topic` field is what is shown in the "Guidelines" section within the Discord client.
-- The `rate_limit_per_user` field limits how frequently threads can be created. There is a new `default_thread_rate_limit_per_user` field on forums as well, which limits how often messages can be sent _in a thread_. This field is copied into `rate_limit_per_user` on the thread at creation time.
+- The `rate_limit_per_user` field limits how frequently threads can be created. There is a new `default_thread_rate_limit_per_user` field on thread-only channels as well, which limits how often messages can be sent _in a thread_. This field is copied into `rate_limit_per_user` on the thread at creation time.
```

Comment:
The create guild channel endpoint respects `default_thread_rate_limit_per_user` for media channels (as well as announcement channels and forums channels). Perhaps this should be documented in the JSON parameters of this endpoint and the channel types should be noted in the channel structure.

Reply:
* #5978